hs_core/models.py

```diff
@@ -3418,7 +3418,7 @@ def size(self):
         """
         # trigger file size read for files that haven't been set yet
         for f in self.files.filter(_size__lt=0):
-            f.calculate_size()
+            f.size()
```

Comment:
I am not sure why we make this line change. Can you clarify a bit? @sblack-usu

Reply:
it's not needed now that I look at it more carefully.  calculate_size resets the db _size value with what is on disk.  size, only sets the db _size value if it hasn't been set.  I wasn't considering the filter above when I made the change.  https://beta.hydroshare.org/resource/d7d98390ab884eeda89f4a10b072bbc3/ was timing out the first couple times I loaded it.  That was only because the _size values hadn't been set yet, not because of my change to size() method.  I'll revert this change.